extensions/wikia/WallNotifications/WallNotifications.class.php

```diff
@@ -464,9 +479,41 @@ protected function sendEmail( User $watcher, array $data ) {
 		return $watcher->sendMail( $data['$MAIL_SUBJECT'], $text, $from, $replyTo, 'WallNotification', $html );
 	}
 
+	protected function getEmailExtensionController( WallNotificationEntity $notification, $watcherName ) {
+		if ( !empty( $notification->data->article_title_ns )
+			&& MWNamespace::getSubject( $notification->data->article_title_ns ) == NS_WIKIA_FORUM_BOARD
+			&& $notification->isMain()
+		) {
+			$controller = 'Email\Controller\Forum';
+		} else if ( !$notification->isMain() ) {
+			$controller = 'Email\Controller\ReplyWallMessage';
+		} else if ( $notification->data->wall_username == $watcherName ) {
+			$controller = 'Email\Controller\OwnWallMessage';
+		} else {
+			$controller = 'Email\Controller\FollowedWallMessage';
+		}
+
+		return $controller;
+	}
+
+	/**
+	 * Get abstract from whole entity text.
+	 * HTML tags are stripped out, expect <p> <br> which are used to make it easier to read.
```

Comment:
Should probably be "HTML tags are stripped out, *except* \<p\> \<br\> which are used to make it easier to read."